packages/types/src/asset-types.ts

```diff
@@ -191,7 +191,11 @@ export type MinimalAsset = {
    */
   coinGeckoId: string | undefined;
   coinName: string;
+  /** Transfers are allowed, but unstable. */
   isUnstable: boolean;
+  /** Transfers are NOT allowed. */
+  isDisabled: boolean;
```

Comment:
What are your thoughts on renaming this to `areTransfersDisabled` instead? The term `isDisabled` is too broad and could imply that the asset can't be selected or skipped altogether